cmd/ps.go

```diff
@@ -53,7 +53,7 @@ func PsScale(appID string, targets []string) error {
 				return err
 			}
 		} else {
-			fmt.Printf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
+			return fmt.Errorf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
```

Comment:
not a global var, a global type since this has variable data in it:

```go
type errInvalidScalePattern struct {
  target string
}

// Error implements the error interface
func (e errInvalidScalePattern) Error() string {
  return fmt.Sprintf("'%s' does not match the pattern 'type=num', ex: web=2\n", e.target)
}
```